setup.py

```diff
@@ -2,31 +2,65 @@
 # -*- coding: utf-8 -*-
 import ast
 import codecs
-import os
+import os.path
 import re
+import subprocess
 import sys
+from codecs import open
+from distutils import log
+from distutils.errors import DistutilsError
 
 from setuptools import find_packages, setup
 from setuptools.command.install import install
+from setuptools.command.sdist import sdist as BaseSDistCommand
 from setuptools.command.test import test as TestCommand
 
-HERE = os.path.abspath(os.path.dirname(__file__))
-init = os.path.join(HERE, "src", "unicef_vision", "__init__.py")
-
+ROOT = os.path.realpath(os.path.dirname(__file__))
+init = os.path.join(ROOT, "src", "unicef_vision", "__init__.py")
 _version_re = re.compile(r'__version__\s+=\s+(.*)')
 _name_re = re.compile(r'NAME\s+=\s+(.*)')
 
+sys.path.insert(0, os.path.join(ROOT, 'src'))
+
 with open(init, 'rb') as f:
     content = f.read().decode('utf-8')
     VERSION = str(ast.literal_eval(_version_re.search(content).group(1)))
     NAME = str(ast.literal_eval(_name_re.search(content).group(1)))
 
 
 def read(*files):
-    content = ''
+    content = []
     for f in files:
-        content += codecs.open(os.path.join(HERE, f), 'r').read()
-    return content
+        content.extend(codecs.open(os.path.join(ROOT, 'src', 'requirements', f), 'r').readlines())
+    return "\n".join(filter(lambda l:not l.startswith('-'), content))
+
+
+def check(cmd, filename):
+    out = subprocess.run(cmd, stdout=subprocess.PIPE)
```

Comment:
I think we should add docstrings for all these helper functions

Reply:
I think we are gonna move to this PR https://github.com/unicef/unicef-vision/pull/7/files
I asked to use simple things